src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -94,6 +94,7 @@
     <Compile Include="$(PathToMsalSources)\Platforms\Features\OpenTelemetry\**\*.cs" />
 
     <PackageReference Include="System.Diagnostics.DiagnosticSource" />
+    <PackageReference Include="System.Formats.Asn1" />
```

Comment:
This is how you add a package ref conditionally. Notice the ItemGroup parent and the condition it has.